src/ui/public/documentation_links/documentation_links.js

```diff
@@ -13,6 +14,9 @@ export const documentationLinks = {
     startup: `${ELASTIC_WEBSITE_URL}guide/en/beats/filebeat/${DOC_LINK_VERSION}/filebeat-starting.html`,
     exportedFields: `${ELASTIC_WEBSITE_URL}guide/en/beats/filebeat/${DOC_LINK_VERSION}/exported-fields.html`
   },
+  metricbeat: {
+    base: `${ELASTIC_WEBSITE_URL}guide/en/beats/metricbeat/${DOC_LINK_VERSION}/`
```

Comment:
Can we have these without the trailing `/`? That composes nicer, for example: `{config.docs.beats.filebeat}/metricbeat-getting-started.html` versus `{config.docs.beats.filebeat}metricbeat-getting-started.html`. It's also more consistent with how we do it in the asciidocs (at least in Beats).

Reply:
>  It's also more consistent with how we do it in the asciidocs (at least in Beats).

Unfortunately, that's going to change at some point because we are standardizing on shared attributes in asciidoc. We'll need to clean up Beats to use https://github.com/elastic/docs/blob/master/shared/attributes.asciidoc... It's just not high on the priority list atm.

Reply:
Ok, but those are still without the `/` from what I see, right?